Formula/afflib.rb

```diff
@@ -31,7 +31,7 @@ class Afflib < Formula
   end
 
   def install
-    ENV["PYTHON"] = Formula["python@3.9"].opt_bin/"python3"
+    ENV["PYTHON"] = Formula["python@3.10"].opt_bin/"python3"
```

Comment:
```suggestion
    ENV["PYTHON"] = which("python3")
```
This will make it easier to switch this formula in the future.